framework/core/locale/core.yml

```diff
@@ -59,6 +59,14 @@ core:
       title: Dashboard
       tools_button: Tools
 
+    # These translations are usin in the debug warning widget.
+    debug-warning:
+      detail: |
+        When <code>debug</code> mode is active, Flarum will rebuild its <code>JavaScript</code> and <code>CSS</code> assets on every request, and could also potentially leak other information, such as database secrets, environment variables, etc.
+
+        It is highly recommended to disable <code>debug</code> in your <code>config.php</code> file in production.
```

Comment:
Do we have a docs page we could link to about this, just for clarification?

Reply:
We have https://docs.flarum.org/troubleshoot/#step-0-activate-debug-mode

Perhaps this is what you had in mind?

Reply:
Yeah, I think that works.